errors.go

```diff
@@ -261,3 +261,49 @@ func decodeInternalError(target interface{}) error {
 type HTTPErrorHandler interface {
 	HandleHTTPError(ctx context.Context, err error, w http.ResponseWriter)
 }
+
+// HTTPStatusCodeToErrorCode maps a http status code integer to an
+// influxdb error code string.
+func HTTPStatusCodeToErrorCode(statusCode int) string {
```

Comment:
did you consider moving this to `kit/http`? imo, this doesn't belong here in the domain. its a http/grpc specific concern. 

the API responder type already does this here: https://github.com/influxdata/influxdb/blob/svc_def/kit/transport/http/api.go#L90

derived from: https://github.com/influxdata/influxdb/blob/svc_def/kit/transport/http/api.go#L232

Reply:
Awesome, cheers @jsteenb2 I will do that. I wasn't sure if it was a little too HTTP specific for the top-level. I will move it.

Reply:
This is related @GeorgeMac https://github.com/influxdata/influxdb/issues/17889